packages/modules/viewer-react/src/components/app-ui/providers/ViewerContentToolsProvider.tsx

```diff
@@ -16,7 +16,16 @@ import * as React from "react";
 
 export class ViewerContentToolsProvider extends StandardContentToolsProvider {
   constructor(private _defaultItems?: DefaultContentTools) {
-    super("ViewerDefaultContentTools", _defaultItems);
+    super(
```

Comment:
gah forgot these extended the Providers from core
imo its probably more of a bug on the core side, 


for content tools youre missing defaults for the horizontal tools as well

i think a cleaner way to do something like this:
<img width="694" alt="image" src="https://user-images.githubusercontent.com/11051042/171883684-ef31cff1-1250-4e72-83ae-60cfbbf4d8bd.png">

Reply:
I think I tried setting them to true initially, but that doesn't work because it overrides any false values passed in through the viewer component:
![image](https://user-images.githubusercontent.com/51834103/171904042-3d857636-ed89-4d0f-acf1-698b7ca14c48.png)
Adding the rest operator also causes the bug for some reason, so I had to remove it.

Reply:
Well you want to spread after you set the default, I think im spreading at the wrong level in the above screenshot